src/app/modules/level-select/level-select-routing.module copy.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+const routes: Routes = [];
+
+@NgModule({
+  imports: [RouterModule.forChild(routes)],
+  exports: [RouterModule]
+})
+export class LevelSelectRoutingModule { }
```

Comment:
As I wrote above, this thing is redundant)

Reply:
Ok I will delete this